worker_isolate.dart

```diff
@@ -15,7 +15,9 @@ import 'package:sintr_common/bucket_utils.dart';
 import 'package:sintr_common/configuration.dart' as config;
 import 'package:sintr_common/logging_utils.dart' as log;
 
-import 'package:smart/completion_model/analyse_path.dart';
+import 'package:smart/completion_model/analyse_path.dart' as completion_model;
+import 'package:smart/discovery_model/analyse_path.dart' as discovery_model;
+
```

Comment:
ack :)

Thanks for the comments @emsod, and yes, this code was somewhat optimized for velocity of getting the experiment working, esp. this file which is complicated by the distributed compute infrastructure it interacts with.

Now you're taking a look through the code, I'll do a pass adding the necessary references to at least make it comprehensible rather than somewhat mystical as it currently stands.